pages/api/search/users/[...action].ts

```diff
@@ -34,11 +42,24 @@ async function post(req: NextApiRequest, res: NextApiResponse) {
       where: whereCondition[action]
     };
 
-    const users = await models.user.findAll(paginate(queryOptions, req.body));
+    const token = (await getToken({ req, secret: serverRuntimeConfig?.auth?.secret })) as JwtToken;
+
+    let scope = "defaultScope";
+
+    const isAdmin = UserRoleUtils.hasAdminRole(token);
+    const isGovernor = UserRoleUtils.hasGovernorRole(token);
+    const isSameUser = lowerCaseCompare(token?.address, req?.body[0]) || lowerCaseCompare(token?.login, req?.body[1]);
+
+    if (isAdmin)
+      scope = "admin";
+    else if (isGovernor || isSameUser)
+      scope = "ownerOrGovernor";
```

Comment:
why is "same user" and "governor" the same thing?

Reply:
@moshmage because both have the same visibility in email columns